filter_plugins/openshift_master.py

```diff
@@ -9,7 +9,7 @@
 import yaml
 
 from ansible import errors
-from ansible.runner.filter_plugins.core import bool as ansible_bool
+from ansible.plugins.filter.core import bool as ansible_bool
```

Comment:
We probably need to figure out a way to handle this import for both 1.9.x and 2.0